test/run.sh

```diff
@@ -102,7 +102,8 @@ go install github.com/jstemmer/go-junit-report/v2@latest
 
 if ! npm list -g xunit-viewer; then npm install -g xunit-viewer; fi
 
-mkdir -p "${THIS_DIR}/results"
+RESULTS_DIR="${THIS_DIR}/results/$(date +%Y-%m-%d-%H-%M-%S)"
```

Comment:
Cool, I think this will still work nicely for https://github.com/gitpod-io/gitpod/blob/main/.github/workflows/ide-integration-tests.yml#L166, and be easier to read, thanks @WVerlaek !

cc: @akosyakov and @iQQBot for posterity